applications/sdp/gpio/src/backend/backend_mbox.c

```diff
@@ -4,10 +4,11 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#include "../include/main.h"
+#include "backend.h"
 #include <zephyr/drivers/mbox.h>
 
 static const struct mbox_dt_spec rx_channel = MBOX_DT_SPEC_GET(DT_PATH(mbox_consumer), rx);
+static backend_callback_t cbck;
```

Comment:
I would rename it to something more descriptive, like `rcv_cb`. Same goes for the `backend_icmsg.c` file.